src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -39,38 +39,44 @@ public AppDomainTests()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #18718")]
```

Comment:
@Jiayili1 this test class is already disabled for Desktop, see the attribute on the class itself: https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Extensions/tests/System/AppDomainTests.cs#L17

So there is no need of adding this attributes to every test, could you please revert your changes to this file?